.github/workflows/eks-ci.yml

```diff
@@ -0,0 +1,134 @@
+
+permissions:
+  actions: read
+  checks: write
+  statuses: read
+
+jobs:
+  Setup:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    timeout-minutes: 15
+    steps:
+    - id: Default
+      env:
+        TEST_TARGETS: ${{ env.TEST_TARGETS_DEFAULT }}
+      run: |
+        echo "TEST_MATRIX=$TEST_TARGETS" >> $GITHUB_ENV
+
+    - id: Dispatch
+      if: github.event_name == 'workflow_dispatch'
+      env:
+        TEST_TARGETS: ${{ github.event.inputs.test-targets }}
+      run: echo "TEST_MATRIX=${TEST_TARGETS}" >> $GITHUB_ENV
+
+    - id: set-matrix
+      run: echo "::set-output name=matrix::${{ env.TEST_MATRIX}}"
+
+  Test_Matrix:
+    needs: ['Setup']
+    strategy:
+      fail-fast: false
+      matrix:
+        # test-target: ${{fromJson(needs.Setup.outputs.matrix)}}
+        include:
+          - test-target: pre_merge
+            range: ''
+            values_file: tests/deploy/values/ci-eks.yml
+            k8s_context: "test-eks-config-1"
+    name: ${{ matrix.test-target }}-tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 90
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      # Setup all required tools
+      - name: Init Hermit
+        run: ./bin/hermit env -r >> $GITHUB_ENV
+
+      - name: Get kubeconfig and select context
+        if: success()
+        run: |
+          aws eks update-kubeconfig --name ${{ matrix.k8s_context }} --region ${{ env.AWS_REGION }} --alias ${{ matrix.k8s_context }}
+          echo 'KUBE_CONFIG_DATA=$(cat ~/.kube/config | base64)' >> $GITHUB_ENV
+          kubectl config use-context ${{ matrix.k8s_context }}
+          kubectl config set-context --current --namespace=${{ env.KUBE_NAMESPACE }}
+
+      - name: Deploy tests Helm chart
+        id: deploy_helm
+        if: success()
+        run: |
+          just deploy-tests-helm ${{ matrix.test-target }} ${{ matrix.values_file }} ${{ matrix.range }}
+
+      - name: Execute tests
+        id: run_tests
+        if: success()
+        run: |
+          just run-tests ${{ matrix.test-target }} ${{ matrix.k8s_context }}
+
+      # - name: Setup tmate session
+      #   uses: mxschmitt/action-tmate@v3
+      #   if: failure()
+      #   with:
+      #     limit-access-to-actor: true
+
+      - name: Copy test results
+        id: copy_results
+        if: always()
+        run: |
+          NODE=$(kubectl get po test-pod-v1 -o jsonpath='{.spec.nodeName}')
+          RESULTS_POD=$(kubectl get po -l k8s-app=eks-results --field-selector spec.nodeName=$NODE -o jsonpath='{.items[*].metadata.name}')
+          kubectl cp $RESULTS_POD:/reports ${{ env.REPORTS_DIR }}
+          kubectl exec $RESULTS_POD -- sh -c 'rm -rf /reports/*'
+
+      # - name: Upload Test Results
+      #   if: success()
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     name: allure_results-${{ matrix.test-target}}
+      #     path: ${{ env.REPORTS_DIR }}
```

Comment:
please remove

Reply:
Uncommented -> required for results publishing.